src/ui/map.js

```diff
@@ -769,6 +778,21 @@ class Map extends Camera {
     }
 
     /**
+     * Returns a Boolean indicating whether a tile is loaded.
+     *
+     * @param {string} sourceId The ID of the source to be checked.
+     * @param {number} tileCoordId The ID of the source to be checked.
+     * @returns {boolean} A Boolean indicating whether the source is loaded.
+     */
+
+    isTileLoaded(sourceId, tileCoordId){
```

Comment:
I think we should hold off making a public `isTileLoaded` method. The number encoding for tile coordinates is a private implementation detail. Also, not all sources have x/y/z tile IDs (GeoJSON, Image, Video, WMS).